megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -347,6 +349,22 @@ public ClientGUI(Client client, MegaMekController c) {
         add(panDisplay, BorderLayout.CENTER);
     }
 
+    private void loadSoundFiles() {
+        if (bingClipChat != null) {
+            bingClipChat.close();
+        }
+        if (bingClipMyTurn != null) {
+            bingClipMyTurn.close();
+        }
+        if (bingClipOthersTurn != null) {
+            bingClipOthersTurn.close();
+        }
```

Comment:
I would recommend adding a space between unrelated if statements in the future, as this can cause misreads (namely that they can be else if statements instead) while parsing the code.